libs/blocks/carousel/carousel.css

```diff
@@ -1,3 +1,7 @@
+.d-none {
+  display: none !important;
```

Comment:
`!important` is heavily discourged.  It breaks the CSS specificity
& it is horrible for maintainability. `!important` is the absolute last solution given a huge list of justifications, not something you just use